packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -243,7 +243,7 @@ if (__DEV__) {
 }
 
 function getOwnerDocumentFromRootContainer(
-  rootContainerElement: Element | Document,
+  rootContainerElement: Element | Document | DocumentFragment,
```

Comment:
Not sure whether this file intentionally inlines the container type or if we should replace this with 
```suggestion
  rootContainerElement: Container,
```
from `ReactDOMHostConfig`